src/screens/Home/ProgressTracker.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { TextBold } from 'assets/fonts/Fonts';
+import { ProgressSteps, ProgressStep } from 'react-native-progress-steps';
+import { Colors } from 'assets/Colors';
+import { View } from 'react-native';
+
+interface TrackerProps {
```

Comment:
Are these being used anywhere, or were you planning to adapt this later? Delete if not relevant